db/schema.rb

```diff
@@ -473,9 +474,91 @@
   add_index "plant_scoring_units", ["scoring_unit_name"], name: "plant_scoring_units_scoring_unit_name_idx", using: :btree
   add_index "plant_scoring_units", ["user_id"], name: "index_plant_scoring_units_on_user_id", using: :btree
 
+  create_table "plant_treatment_types", force: :cascade do |t|
+    t.integer  "parent_ids",              array: true
+    t.string   "name",       null: false
+    t.string   "term",       null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "plant_treatment_types", ["name"], name: "index_plant_treatment_types_on_name", unique: true, using: :btree
+  add_index "plant_treatment_types", ["term"], name: "index_plant_treatment_types_on_term", unique: true, using: :btree
+
+  create_table "plant_trial_environments", force: :cascade do |t|
+    t.integer  "plant_trial_id",               null: false
+    t.float    "day_temperature"
+    t.float    "night_temperature"
+    t.float    "temperature_change"
+    t.float    "ppfd_canopy"
+    t.float    "ppfd_plant"
+    t.float    "light_period"
+    t.float    "light_intensity"
+    t.float    "light_intensity_range"
+    t.float    "outside_light"
+    t.float    "rfr_ratio"
+    t.float    "daily_uvb"
+    t.float    "total_light"
+    t.boolean  "co2_controlled"
+    t.float    "co2_light"
+    t.float    "co2_dark"
+    t.float    "relative_humidity_light"
+    t.float    "relative_humidity_dark"
+    t.float    "rooting_container_volume"
+    t.float    "rooting_container_height"
+    t.integer  "rooting_count"
+    t.float    "sowing_density"
+    t.float    "soil_porosity"
+    t.float    "soil_penetration"
+    t.float    "soil_organic_matter"
+    t.float    "medium_temperature"
+    t.float    "water_retention"
+    t.float    "nitrogen_concentration_start"
+    t.float    "nitrogen_concentration_end"
+    t.float    "conductivity"
+    t.datetime "created_at",                   null: false
+    t.datetime "updated_at",                   null: false
+  end
+
+  add_index "plant_trial_environments", ["plant_trial_id"], name: "index_plant_trial_environments_on_plant_trial_id", using: :btree
+
+  create_table "plant_trial_topological_descriptors", force: :cascade do |t|
```

Comment:
There is no such convention, that's just how I usually do it. I don't often use HABTM relationships so it was not a problem for me yet. If we skip the namespace part it is also possible to have a collision in case we want to put similar model into another namespace (e.g. `SomethingElse::TopologicalDescriptor`).